example-workflows/gke-kustomize/.github/workflows/gke-kustomize.yml

```diff
@@ -28,6 +28,10 @@ env:
   REPOSITORY: samples # TODO: update to Artifact Registry docker repository
   IMAGE: static-site
 
+permissions:
```

Comment:
It'd be better to put this inside the specific job that needs it, that way it's not exposed in all jobs.